kubeflow/tf-job/prototypes/tf-cnn-benchmarks.jsonnet

```diff
@@ -62,39 +65,40 @@ local numWorkers = import 'param://num_workers';
 local numGpus = import 'param://num_gpus';
 
 local workerSpec = if numGpus > 0 then
-  	tfJob.parts.tfJobReplica("WORKER", numWorkers, args, imageGpu, numGpus)
-  	else
-  	tfJob.parts.tfJobReplica("WORKER", numWorkers, args, image);
+  tfJob.parts.tfJobReplica("WORKER", numWorkers, args, imageGpu, numGpus)
+else
+  tfJob.parts.tfJobReplica("WORKER", numWorkers, args, image);
 
 // TODO(jlewi): Look at how the redis prototype modifies a container by
 // using mapContainersWithName. Can we do something similar?
 // https://github.com/ksonnet/parts/blob/9d78d6bb445d530d5b927656d2293d4f12654608/incubator/redis/redis.libsonnet
-local replicas = std.map(function(s)   
-  s + {
-    template+: {
-      spec+:  {
-        // TODO(jlewi): Does this overwrite containers? 
-        containers: [
-          s.template.spec.containers[0] + {
-            workingDir: "/opt/tf-benchmarks/scripts/tf_cnn_benchmarks",
-          },
-        ]
-      }
-    },
-  },
-  std.prune([workerSpec, tfJob.parts.tfJobReplica("PS", numPs, args, image)]));
+local replicas = std.map(function(s)
+                           s {
```

Comment:
this looks too much indent?

Reply:
Reran the tool. This looks like the expected output. I think its indented 2 spaces relative to function because its continuing function.